util/network-devp2p/src/discovery.rs

```diff
@@ -389,14 +382,14 @@ impl<'a> Discovery<'a> {
 		node.endpoint.to_rlp_list(&mut rlp);
 		append_expiration(&mut rlp);
 		let old_parity_hash = keccak(rlp.as_raw());
-		let hash = self.send_packet(PACKET_PING, &node.endpoint.udp_address(), &rlp.drain())?;
+		let hash = self.send_packet(PACKET_PING, node.endpoint.udp_address(), &rlp.drain())?;
 
 		self.in_flight_pings.insert(node.id, PingRequest {
 			sent_at: Instant::now(),
 			node: node.clone(),
 			echo_hash: hash,
 			deprecated_echo_hash: old_parity_hash,
```

Comment:
Is there a chance we can get rid of this you think?

Reply:
what do you mean?

Reply:
No, I mean `deprecated_echo_hash` – I guess we must have made a change here at some point and kept the old hash around for backwards compatibility. Do we still have to include it you think? Can we ever remove it?

Reply:
Probably but I'm not sure.